### PR TITLE
DOC-2599: Updated the demo content and configurations as requested.

### PR DESCRIPTION
**Pull request by abhinavgandham**
Ticket: DOC-2599

Site: [Staging branch](http://docs-hotfix-7-doc-2599.staging.tiny.cloud/docs/tinymce/latest/uploadcare/#interactive-example)

Changes:
* Updated the content and configurations for the Uploadcare Image Optimizer demo.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Comment by abhinavgandham** [comment]
@FarzadHayat Yeah sounds like a good idea, however the above code snippet is missing the `uploadcare_cdn_base_url`, is this intentional?

**Review by kimwoodfield — APPROVED** [review]

File: modules/ROOT/examples/live-demos/uploadcare/index.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 tinymce.init({
   selector: "textarea",
   plugins: [ "uploadcare", "code", "link", "preview", "lists" ],
-  uploadcare_public_key: 'cfef242412638bfc4193',	
+  uploadcare_public_key: '630992ad50fe2291c406',
```
Comment (tiny-ben-tran) [inline modules/ROOT/examples/live-demos/uploadcare/index.js:4]: Just double check that it is ok to expose public key?

**Review by tiny-ben-tran — APPROVED** [review]

**Review by shanmen-tiny — APPROVED** [review]

File: modules/ROOT/examples/live-demos/uploadcare/index.html
[the remark this answers is not in the document]
Comment (ShiridiGandham) [reply modules/ROOT/examples/live-demos/uploadcare/index.html:1]: @FarzadHayat 
I’d like to double-check with Product (Fredrik/Kate) before making any changes to the image

@abhinavgandham 
Please hold off for now and wait for further instructions. I’ll get back to you shortly.

[the remark this answers is not in the document]
Comment (ShiridiGandham) [reply modules/ROOT/examples/live-demos/uploadcare/index.html:1]: @abhinavgandham Please proceed with the content from Fiddle as it is. There's no need to change the image URLs.

**Review by ShiridiGandham — APPROVED** [review]
LGTM.

File: modules/ROOT/examples/live-demos/uploadcare/example.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+tinymce.init({
+    selector: "textarea",
+    plugins: [ "uploadcare", "code", "link", "preview", "lists" ],
+    uploadcare_public_key: '<your-public-key>',
+    uploadcare_store_type: 'temporary',
```
Comment (spocke) [inline modules/ROOT/examples/live-demos/uploadcare/example.js:5]: It's not likely that customers will configure this so maybe we skip that from being in the demo if they want to change the default for some odd reason then they can look that up in the docs.

File: modules/ROOT/examples/live-demos/uploadcare/index.html [hunk 3]
```diff
@@ -1,86 +1,33 @@
...
+<figure class="image">
+   <img style="width: min(2048px, 100%); aspect-ratio: 2048 / 718;" src="https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/preview/" sizes="(min-width: 2048px) 2048px, 100vw" srcset="https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/100x/ 100w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/200x/ 200w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/300x/ 300w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/500x/ 500w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/750x/ 750w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/1000x/ 1000w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/1250x/ 1250w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/1500x/ 1500w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/resize/2000x/ 2000w,https://ucarecdn.com/b51db038-86df-4a83-bd9d-6942151306d0/-/preview/ 2048w" alt="Mallorca">
```
Comment (spocke) [inline modules/ROOT/examples/live-demos/uploadcare/index.html:5]: - [ ] We talked about changing these to tiny.ucarecdn.com urls to be consistent.

**Review by spocke — APPROVED** [review]

File: modules/ROOT/examples/live-demos/uploadcare/index.html (hunk 3 above)
Comment (abhinavgandham) [reply modules/ROOT/examples/live-demos/uploadcare/index.html]: The latest commit has the updated URLs. So this is resolved.

File: modules/ROOT/examples/live-demos/uploadcare/example.js (hunk 2 above)
Comment (abhinavgandham) [reply modules/ROOT/examples/live-demos/uploadcare/example.js:5]: Yeah agreed, I will remove from the example.js file.

**Review by ShiridiGandham — APPROVED** [review]
Demo works fine with updated configs and content.

**Pull request merged**